libyul/optimiser/DataFlowAnalyzer.cpp

```diff
@@ -391,53 +401,27 @@ bool DataFlowAnalyzer::inScope(YulString _variableName) const
 }
 
 std::optional<pair<YulString, YulString>> DataFlowAnalyzer::isSimpleStore(
-	evmasm::Instruction _store,
```

Comment:
Can you remove the includes for `libevmasm` then? Or is it still needed?

Reply:
There was even a fat ``TODO avoid`` comment there and I still missed it :-). Removed it now (and the EVMDialect include in the implementation as well).